docs/api/taxons.rst

```diff
@@ -0,0 +1,760 @@
+                        "self": {
+                            "href": "/api/v1/taxons/8"
+                        }
+                    }
+                }
+            ]
+        }
+    }
+
+.. note::
+
+    The images should be passed in array as an attribute (files) of request. See how it is done in Sylius
+    `here <https://github.com/Sylius/Sylius/blob/master/tests/Controller/TaxonApiTest.php>`_.
+
+Updating Taxon
+--------------
+
+To fully update a taxon you will need to call the ``/api/v1/taxons/taxon_id`` endpoint with ``PUT`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    PUT /api/v1/taxons/{id}
+
++------------------------------------+----------------+--------------------------------------+
+| Parameter                          | Parameter type | Description                          |
++====================================+================+======================================+
+| Authorization                      | header         | Token received during authentication |
++------------------------------------+----------------+--------------------------------------+
+| id                                 | url attribute  | Id of the requested taxon            |
++------------------------------------+----------------+--------------------------------------+
+|translations['locale_code']['name'] | request        | Name of the taxon                    |
++------------------------------------+----------------+--------------------------------------+
+|translations['locale_code']['slug'] | request        | **(unique)** Slug                    |
++------------------------------------+----------------+--------------------------------------+
+
+Example
+^^^^^^^
+
+To full update the taxon with ``id = 8`` use the below method:
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/taxons/8 \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Content-Type: application/json" \
+        -X PUT \
+        --data '
+            {
+                "translations": {
+                    "en_US": {
+                        "name": "Dolls",
+                        "slug": "dolls"
+                    }
+                }
+	        }
+        '
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 204 No Content
+
+To update a taxon partially you will need to call the ``/api/v1/taxons/taxon_id`` endpoint with the ``PATCH`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    PATCH /api/v1/taxons/{id}
+
++---------------+----------------+--------------------------------------+
+| Parameter     | Parameter type | Description                          |
++===============+================+======================================+
+| Authorization | header         | Token received during authentication |
++---------------+----------------+--------------------------------------+
+| id            | url attribute  | Id of the requested taxon            |
```

Comment:
We should add all possible parameters here. Copy paste from creating action(almost, without code AFAIR)